### ISSUE TITLE
How can I make a tile-based floor flat? 

### ISSUE THREAD

**Issue by stfnnklc** [liabru/matter-js#937]
Hi,

How can I make a tile-based floor behave as if it were a long rectangle? 

https://codepen.io/stfnnklc/pen/KKMLjrY
Notice that the blue square almost always stops right above a tile.   

Thanks in advance,
Stefan

**Comment by codylico** [comment]
you may want to see liabru/matter-js#745.

**Comment by stfnnklc** [comment]
> you may want to see #745.

Thank you for the answer.
Merging them together would be a bit complicated, because I need the tiles to move independently, but if some of them are inline I want them to behave as if they were merged.
If possible, I would like to avoid recalculating what gets to be merged/unmerged on every frame. 
I tried to make the blue square rounded, but that makes it jump whenever it hits the edge of a tile.  

Is there a way to make the simulation less precise? Maybe that would make it ignore the edges.
